UnitClasses/Paladin.lua

```diff
@@ -44,7 +44,7 @@ function LightOfDawn:Compute()
 
     return {
         heal = heal,
-        aoeHpm = self:GetValPerMana(heal * 3)
+        aoeHpm = self:GetValPerMana(heal * PHD.AOE_AVERAGE_TARGETS)
```

Comment:
Can you replace this for the other classes as well?

Reply:
Good idea. I was leaving most Druid stuff to you, but sure, I will fix in next PR!